components/04-reference-pages/07-faculty-profile-page/faculty-profile-page.hbs

```diff
@@ -54,98 +56,140 @@
                   <h5>Degree</h5>
                   <ul>
                     <li>Ph.D. The University of Texas at San Antonio</li>
-                    <li>M.S. St. Mary’s University</li>
-                    <li>MBA St. Mary’s University</li>
-                    <li>BBA St. Mary’s University</li>
+                    <li>M.S. St. Mary's University</li>
+                    <li>MBA St. Mary's University</li>
+                    <li>BBA St. Mary's University</li>
                   </ul>
                 </div>
               </div>
             </div>
             <!--End Contact Details -->
+
+            <div class="col-xxl-12 col-xl-12 col-lg-12 col-md-12 col-sm-12">
```

Comment:
**div class="col-12"**  is all that is necessary here.

Reply:
👍